### PR TITLE
Test service catalog after appliance update

### PR DESCRIPTION
**Pull request by sshveta**
Test service catalog after appliance update

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 11422c8a2150db61bce9764938add42a97244b25:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit ada19207fd96c3eec7e9fcc83a17ed601d40d3f7:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by mshriver** [comment]
@sshveta Please address ssh timeouts in PRT run 8046

File: cfme/tests/services/test_service_catalog_after_update.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+@pytest.mark.tier(2)
+def test_service_catalog_after_update(update_appliance, provider, setup_provider,
+                                      catalog_item, request):
+    """Tests order catalog item
+    Metadata:
+        test_flag: provision
+    """
+    vm_name = catalog_item.provisioning_data["vm_name"]
+    request.addfinalizer(lambda: cleanup_vm(vm_name + "_0001", provider))
```
Comment (psav) [inline cfme/tests/services/test_service_catalog_after_update.py]: https://youtu.be/wD6dqm9hSRM?t=42m57s

**Comment by sshveta** [comment]
Closing pull request as it cannot be run on jenkins

**Pull request closed without merging**